src/app/dashboard/dl/DigitalDashboard.tsx

```diff
@@ -0,0 +1,190 @@
+import { api } from "techme/trpc/react";
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  ArcElement,
+  Title,
+  Tooltip,
+  Legend,
+);
+
+const DigitalDashboard = () => {
+  const { data: projects } = api.projects.getMyProjectsStatus.useQuery();
+  const { data: projectsDays, isLoading } =
+    api.projects.getMyProjectsDeadline.useQuery();
+
+  return (
+    <div className="grid grid-cols-1 gap-6 p-6 md:grid-cols-2 lg:grid-cols-3">
+      {/* Pending Projects */}
+      <Card className="rounded-2xl shadow-lg transition-shadow hover:shadow-2xl">
+        <CardHeader>
+          <CardTitle>
+            <FaProjectDiagram className="mr-2 inline-block" />
+            Pending Projects
+          </CardTitle>
+          <CardDescription>
+            {projectsDays?.length ?? 0} projects
+          </CardDescription>
+        </CardHeader>
+        <CardContent>
+          <div className="space-y-2">
+            {isLoading ? (
+              <p>Loading...</p>
+            ) : (
+              projectsDays?.map((projectDays) => (
+                <div key={projectDays.id}>
+                  <p className="text-lg font-medium">{projectDays.name}</p>
+                  <p
+                    className={`text-lg ${
+                      typeof projectDays.days_left === "number" &&
+                      projectDays.days_left <= 1
+                        ? "text-red-500"
+                        : "text-yellow-500"
+                    }`}
+                  >
+                    {String(projectDays.days_left)}{" "}
+                    {projectDays.days_left === 1 ? "Day" : "Days"} Left
+                  </p>
+                </div>
+              ))
+            )}
+          </div>
+        </CardContent>
+        <CardFooter>
+          <Button variant="link" className="flex items-center space-x-2">
+            <FaTasks />
+            <span>View Details</span>
+          </Button>
+        </CardFooter>
+      </Card>
+
+      {/* Analyze Projects */}
+      <Card className="rounded-2xl shadow-lg transition-shadow hover:shadow-2xl">
+        <CardHeader>
+          <CardTitle>
+            <FaChartPie className="mr-2 inline-block" />
+            Analyze Projects
+          </CardTitle>
+          <CardDescription>View project analytics</CardDescription>
+        </CardHeader>
+        <CardContent>
+          <Button className="rounded px-4 py-2 text-white">Analyze</Button>
+        </CardContent>
+      </Card>
+
+      {/* View Team */}
+      <Card className="rounded-2xl shadow-lg transition-shadow hover:shadow-2xl">
+        <CardHeader>
+          <CardTitle>
+            <FaUsers className="mr-2 inline-block" />
+            View Team
+          </CardTitle>
+          <CardDescription>Manage your team</CardDescription>
+        </CardHeader>
+        <CardContent>
+          <Button
+            className="rounded px-4 py-2 text-white"
+            onClick={() => {
+              window.location.href = "/team";
```

Comment:
Use the nextjs navigation, https://nextjs.org/docs/app/building-your-application/routing/linking-and-navigating#userouter-hook